bigquery_schema_generator/generate_schema.py

```diff
@@ -62,7 +62,7 @@ class SchemaGenerator:
     TIME_MATCHER = re.compile(r'^\d{1,2}:\d{1,2}:\d{1,2}(\.\d{1,6})?$')
 
     # Detect integers inside quotes.
-    INTEGER_MATCHER = re.compile(r'^[-]?\d+$')
+    INTEGER_MATCHER = re.compile(r'^-?\d+\.?(?:\d+[e|E]\+\d+)?$') # match scientific notation as well as (^-?\d+$)
```

Comment:
Please send this change as a separate PR.

Reply:
#57 has been merged as #62. Please rebase on to `develop` and update this PR.
With regards to the floating point in quotes, I don't think I understand the problem.

Reply:
Python's JSON parser comprehends scientific notation but thinks that it must be a float, even for the integers like "1.43e5", hence the forced is_integer() check.

Reply:
I see. I think you correctly point out that `INTEGER_MATCHER` and `FLOAT_MATCHER` do not accept all forms of integers and floats that `bq load` accepts. However, I think you are mistaken about "1.43e5" being an integer, because `bq load --autodetect` infers that type as a float.

I submitted a commit that updates `INTEGER_MATCHER` and `FLOAT_MATCHER` to be the following:

```python
INTEGER_MATCHER = re.compile(r'^[-+]?\d+$')
FLOAT_MATCHER = re.compile(r'^[-+]?(?:\d+\.?\d*|\.\d+)(?:[eE][-+]?\d+)?$') 
```

I added unit tests, and ran a bunch of experiments of `bq load --autodtect` and these seem to match the behavior of `bq load`.

Maybe you can pull the latest version from `develop` and let me know if it works for you.

Reply:
I released these changes as v1.4 and pushed to PyPI. I am inclined to close this PR.